src/Compilers/CSharp/CSharpAnalyzerDriver/CSharpDeclarationComputer.cs

```diff
@@ -215,6 +228,8 @@ private static bool InvalidLevel(int? level)
                             codeBlocks = codeBlocks.Concat(expressionBody);
                         }
 
+                        codeBlocks = codeBlocks.Concat(GetAttributes(t.AttributeLists));
```

Comment:
>codeBlocks = codeBlocks.Concat(GetAttributes(t.AttributeLists)); [](start = 24, length = 64)

What about attributes on type parameters?

Reply:
[https://github.com/dotnet/roslyn/issues/23348](https://github.com/dotnet/roslyn/issues/23348)

---
In reply to: [152412100](https://github.com/dotnet/roslyn/pull/23318#discussion_r152412100) [](ancestors = 152412100)

Reply:
Addressed.